src/main/scala/org/constellation/domain/transaction/TransactionChainService.scala

```diff
@@ -12,6 +12,9 @@ class TransactionChainService[F[_]: Concurrent] {
   def getLastTransactionRef(address: String): F[LastTransactionRef] =
     lastTransactionRef.get.map(_.getOrElse(address, LastTransactionRef.empty))
 
+  def setLastTransactionTemporary(tx: Transaction): F[LastTransactionRef] =
```

Comment:
rm

Reply:
Once you will confirm that those changes are not breaking the cluster we can enable it but just reverting/removing 1 commit.

Reply:
I have removed it so the PR can be merged